packages/api/src/clickhouse/index.ts

```diff
@@ -1130,8 +1138,14 @@ const buildEventSeriesQuery = async ({
   const label = SqlString.escape(`${aggFn}(${field})`);
 
   const selectClause = [
-    isCountFn
+    aggFn === AggFn.Count
       ? 'toFloat64(count()) as data'
+      : aggFn === AggFn.CountPerSec
+      ? "divide(count(), age('ss', MIN(timestamp), MAX(timestamp))) as data"
```

Comment:
shouldn't it be divided by the diff of the time range given by the user? (I noticed this issue from the test values showing 0.75 for the rate instead of what I'd calculate by hand to be 0.6)

Reply:
That was my original implementation. I was debating which version is what users expect to see actually.

Reply:
got the idea. going to update the code